docs/artifact_repositories/api.md

```diff
@@ -1,85 +1,39 @@
 # API
 
-This section documents the public API for Phylum's artifact repository offering.
+This section provides additional instructions for usage of Phylum's Artifact
+Repository API. The full API is documented on [Swagger].
 
-## Webhooks
+[Swagger]: https://swagger.phylum.io/?urls.primaryName=Artifact+Repository+API
 
-Webhooks will send out notifications whenever a package that was accessed
-through a repository proxy fails analysis.
+## Authentication
 
-To setup a new webhook, you can use the following curl command:
+The Artifact Repository API expects authentication in the basic auth format. The
+username is optional and provides context for which group should be used for the
+request. The group itself must either be the group name itself, for standalone
+groups, or the organization and group name combined using a `/`.
 
-```sh
-# Phylum group used with the registry proxy.
-export PHYLUM_GROUP=…
-# https://docs.phylum.io/knowledge_base/api-keys#generate-an-api-key
-export PHYLUM_API_KEY=…
-# URL which will be called on analysis failure.
-export WEBHOOK_URL=…
-# Shared secret used to validate authenticity of webhook callbacks
-export SECRET=…
+The following script will create a valid token for the org `phylum`, group
+`demo`, and with the token stored in `$PHYLUM_API_KEY`:
```

Comment:
```suggestion
The following script will create a properly formatted authorization header for
the org `phylum`, group `demo`, and with a valid token stored in
`$PHYLUM_API_KEY`:
```
This is just one suggested alternative, but the script does not **create** a valid token; it needs a valid token, formatted properly.

Reply:
It will create a valid authorization token though.